cypress/objects/common.js

```diff
@@ -1,7 +1,27 @@
 class Common {
-    checkIfUrlIncludes(text) {
-        cy.url().should('include', text);
+    checkIfUrlDoesNotInclude(text) {
+        cy.url().should('not.include', text);
     }
+
+    clickCertainButton(buttonType, text) {
+        cy.get(`[type="${buttonType}"]`).contains(text).click();
+    }
+
+    generateAUniqueString() {
```

Comment:
better to call this function getCurrentTimeStamp, as it would be more appropriate name for it